braindecode/models/base.py

```diff
@@ -215,6 +217,18 @@ def get_output_shape(self) -> Tuple[int]:
                     raise ValueError(msg) from exc
                 raise exc
 
+    def load_state_dict(self, state_dict, *args, **kwargs):
+
+        mapping = self.mapping
```

Comment:
```suggestion
    mapping = None
    
    def load_state_dict(self, state_dict, *args, **kwargs):

        mapping = self.mapping if self.mapping else {}
```
So that we don’t have to add this mapping attribute to the models that don’t need if